src/core/muselementfactory.cpp

```diff
@@ -527,6 +527,7 @@ bool CAMusElementFactory::configureFunctionMark( CAFunctionMarkContext *fmc, int
 	Configures a new tuplet containing the given \a noteList.
  */
 bool CAMusElementFactory::configureTuplet( QList<CAPlayable*> noteList ) {
+    (void)noteList;
```

Comment:
I suggest we just use anonymous ` bool CAMusElementFactory::configureTuplet( QList<CAPlayable*> ) {` in the head which gets rid of the unused warnings.

Reply:
Done.